### PR TITLE
fixing https://issues.openmrs.org/browse/FM-187

### PR DESCRIPTION
**Pull request by rasanjanap**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -90,6 +90,30 @@
             <version>1.5</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+        <groupId>org.openmrs.module</groupId>
```
Comment (harsha89) [inline api/pom.xml]: formatting issue

File: api/pom.xml [hunk 2]
```diff
@@ -90,6 +90,30 @@
...
+        <artifactId>appointmentscheduling-api</artifactId>
+        <version>${appointmentschedulingVersion}</version>
+        <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>reporting-api</artifactId>
+            <version>0.9.2</version>
```
Comment (harsha89) [inline api/pom.xml]: versions should go to properties

**Comment by harsha89** [comment]
@surangak can you check the mappings? Current things seems fine. we have some other fields to maps as well.

**Comment by harsha89** [comment]
Please add a test as well

File: api/pom.xml (hunk 1 above)
Comment (rasanjanap) [inline api/pom.xml]: Will address it

**Comment by rasanjanap** [comment]
@harsha89 will do. Is current mappings looks fine?

**Comment by rasanjanap** [comment]
@harsha89 updated pull request. But pull couldn't sorted out the dependency issue

**Comment by harsha89** [comment]
Looks good!

**Comment by dkayiwa** [comment]
@harsha89 can you merge this if it is good?

**Comment by harsha89** [comment]
@dkayiwa we have a talk post regarding this. There are some dependency issues causes test to fail. I too look at it but couldn't get sorted out. Since that we can't merge this until we fix dependency issue

**Comment by dkayiwa** [comment]
@harsha89 so is any one actively working on fixing this? :)

**Comment by harsha89** [comment]
@dkayiwa I had chat with @rasanjanap let's see we can find the cause soon :) I too work on this.

**Comment by dkayiwa** [comment]
@harsha89 did you succeed in this?

**Comment by dkayiwa** [comment]
@rasanjanap and @harsha89 these are the changes you need to make in order to fix the dependency issues https://github.com/openmrs/openmrs-module-fhir/pull/119/commits/c0e0c831e8b1c11bdf6da22008982d26695929ab

Feel free to remove the @Ignore attribute when done with AppointmnetServiceTest

**Comment by harsha89** [comment]
@dkayiwa i have completely missed it. Let me check with @rasanjanap and finish it

**Pull request merged**